modules/ROOT/pages/get-started.adoc

```diff
@@ -4,12 +4,39 @@ After you become familiar with the UI and concepts covered in the xref:index.ado
 
 == Before You Begin
 
-* To access the API Governance console, you must have the *API Governance* > *Governance Administrator* permission.
+Before you begin using API Governance:
 
-* If you have an opt-in banner in the API Governance console, opt in to get additional functionality. See the xref:release-notes::api-governance/api-governance-release-notes.adoc[] for details.
+* <<verify-permissions>>
+* <<opt-in>>
+
+[[verify-permissions]]
+=== Verify Your Permissions
+
+Ensure you have access to API Governance with one of the following permissions:
+
+* *API Governance* > *Governance Viewer*
 +
-If you don't see the opt-in banner, you already have access to the new functionality.
+This permission allows API Governance users, such as API developers and architects, to:
+
+** View the API Governance console conformance summary 
+** View the *Governed APIs* page with the list of governed APIs that they have permission to view
+** Download a detailed governance report for their APIs in CSV format.
++
+* *API Governance* > *Governance Administrator*
 +
+This permission allows API Governance users, such as API governance and security officers, full access to API Governance. 
+
+Anypoint Platform organization administrators are allowed full access to API Governance regardless of their API Governance permission settings. 
+
+The *Governance Viewer* permission is overridden by higher level permissions. If a user has the *Governance Administrator* permission or is an organization administrator, setting the *Governance Viewer* permission for them does not limit their permissions.
+
+[[opt-in]]
```

Comment:
@bjoson-mulesoft governance viewers will not see the banner, therefore they might be in the old experience and not know about it. Not sure if we should document this or add some "banner" 🤔